customer-service/src/main/kotlin/cz/kss/proj/clientservice/handler/CustomerHandler.kt

```diff
@@ -0,0 +1,33 @@
+package cz.kss.proj.clientservice.handler
+
+import cz.kss.proj.clientservice.dto.CreateCustomerDto
+import cz.kss.proj.clientservice.service.CustomerService
+import org.springframework.http.MediaType
+import org.springframework.stereotype.Component
+import org.springframework.web.reactive.function.server.*
+
+@Component
+class CustomerHandler(
+    private val customerService: CustomerService
+) {
+
+    suspend fun createCustomer(req: ServerRequest): ServerResponse {
+        val request = req.awaitBodyOrNull<CreateCustomerDto>()
+        return request?.let {
+            val createCustomer = customerService.createCustomer(request)
```

Comment:
Here comes the case for my custom `Result` type :-)

Reply:
Yes, I count with it